File: app/Models/PeriodeGaji.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class PeriodeGaji extends Model
{
    protected $fillable = ['tanggal_mulai', 'tanggal_selesai'];

    public function slipGajis()
    {
        return $this->hasMany(SlipGaji::class);
    }
}

File: Models/PeriodeGaji.cs

using System.Text.Json.Serialization;

namespace SistemPengirimanApp.Models;

public class PeriodeGaji
{
    public int Id { get; set; }

    [JsonPropertyName("tanggal_mulai")]
    public DateTime TanggalMulai { get; set; }

    [JsonPropertyName("tanggal_selesai")]
    public DateTime TanggalSelesai { get; set; }
}